include/ck_tile/host/arg_parser.hpp

```diff
@@ -187,6 +240,46 @@ class ArgParser
         return value;
     }
 
+    std::vector<std::string> get_string_vec(const std::string& str,
```

Comment:
Please be consistent with other `get_... ` functions. 
```suggestion
    std::vector<std::string> get_string_vec(const std::string& name,
```